testcases/common/common.c

```diff
@@ -1102,34 +1110,81 @@ int do_ParseArgs(int argc, char **argv)
 //
 int do_GetFunctionList(void)
 {
-    CK_RV rc;
+    CK_INTERFACE *interface;
+    CK_VERSION version;
+    CK_FLAGS flags;
+    CK_RV rc, rv;
     CK_RV(*pfoo) ();
     char *e;
     char *f = "libopencryptoki.so";
+    CK_ULONG nmemb = 0;
+
+    rv = FALSE;
```

Comment:
Looks like rv is a CK_RV, not a CK_BBOOL, are you intentionally assigning rv = FALSE?

Reply:
done